apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/item/CIFileView.kt

```diff
@@ -152,10 +152,11 @@ fun CIFileView(
             when (file.fileProtocol) {
               FileProtocol.XFTP -> progressIndicator()
               FileProtocol.SMP -> fileIcon()
+              FileProtocol.LOCAL -> fileIcon()
             }
           is CIFileStatus.SndTransfer ->
             when (file.fileProtocol) {
-              FileProtocol.XFTP -> progressCircle(file.fileStatus.sndProgress, file.fileStatus.sndTotal)
+              FileProtocol.XFTP, FileProtocol.LOCAL -> progressCircle(file.fileStatus.sndProgress, file.fileStatus.sndTotal)
```

Comment:
probably local files should have their own branch and always show empty view in place of status icon (they can't be transferred), and don't show "waiting" alerts programmed (they can't accepted) - here and same in CIImageView, CIVideoView

Reply:
they're always created in sndStored status and if I remember correctly never change state